yield_plugin/src/yield_plugin.cpp

```diff
@@ -427,11 +427,27 @@ namespace yield_plugin
     // Up until goal_pos (which also can be until end of the entire original trajectory), generate new speeds at
     // or near original trajectory points by generating them at a fixed time interval using the JMT polynomial equation
     const double initial_time = 0;
-    const double initial_accel = 0;
+    double initial_accel = 0;
+    if (last_speed_)
```

Comment:
can you check if last_speed_ and last_speed_time_ both have values here?

Reply:
added both